indexBasecodeANSWER1c.html

```diff
@@ -0,0 +1,157 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+		<title>Flights to Mars</title>
+    <link href="design2.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+      <br>
+      <div id="wrapper">
+            <div id="mainphoto">&nbsp;</div>  <!-- end mainphoto -->
+            <div id="nav">
+                  <table id='table1' width='900'>
+                  <tr>
+                  <td align='center' width='110' >
+                  <img src='images\mars.jpg' alt='This is Mars' height='100' id='mars'>
+					  </td>
+                  <td align='center' width='110' >
+                  <img src='images\earth.jpg' alt='This is Earth' height='100' id='earth' >
+                  </td>
+                  <td align='center' width='110' >
+                  <img src='images\pluto.jpg' alt='This is Pluto' height='100' id='pluto' >
+                  </td>
+                  <td align='center' width='110' >
+                  <img src='images\mercury.jpg' alt='This is Mercury' height='100' id='mercury'>
+                  </td>
+                  <td align='center' width='110' >
+                  <img src='images\venus.jpg' alt='This is Venus' height='100' id='venus' >
+                  </td>
+                  <td align='center' width='110' >
+                  <img src='images\moon.jpg' alt='This is Moon' height='100' width='100' id='moon' >
+                  </td>
+                  </tr>
+                  <tr>
+                  <td id='marsT' align='center' width='110' >&nbsp;</td>

```

Comment:
The 'align' and 'width' attributes are repeated for every element, could benefit from defining this in a reusable css style.